docs/sphinx/reference-libobs-util-serializers.rst

```diff
@@ -102,9 +102,11 @@ Provides an output serializer used with dynamic arrays.
 Array Output Serializer Structure (struct array_output_data)
 ------------------------------------------------------------
 
-.. type:: struct array_output_data
+.. struct:: array_output_data
 
-.. member:: DARRAY(uint8_t) array_output_data.bytes
+.. code:: cpp
```

Comment:
Isn't this code technically C rather than C++?

Reply:
Technically, yes, but all other code blocks we have on docs are all `.. code:: cpp` so I did this for consistency.

Reply:
Fair enough.